app/(service)/my-store/[storeId]/_components/myStoreCard.tsx

```diff
@@ -7,56 +7,102 @@ import Link from 'next/link';
-            <div className="font-bold text-[24px] flex gap-2 items-center md:text-[28px]">{storeData.name}</div>
+    <>
+      <div className=" bg-red-10 flex flex-col w-[351px] xl:w-[964px] gap-3 p-5 mx-auto mb-3 md:w-[632px] border rounded-xl border-gray-20 md:p-6 md:mb-6 md:gap-4 xl:flex-row xl:gap-8 ">
+        <div className=" rounded-lg overflow-hidden w-[311px] h-[177px] md:max-w-full md:w-[632px] md:h-[361px] xl:w-[539px] xl:h-[308px]  bg-slate-500 relative">
+          <Image src={storeData.imageUrl} alt="가게이미지" fill priority />
+        </div>
+        <div className="flex flex-col justify-between xl:w-[343px] ">
+          <div className="flex flex-col gap-2 xl:pt-4 md:gap-3">
+            <div>
+              <span className="inline-block mb-2 font-bold text-primary text-[14px] md:text-[16px]">
+                {storeData.category}
+              </span>
+              <div className="font-bold text-[24px] flex gap-2 items-center md:text-[28px]">{storeData.name}</div>
+            </div>
+            <div className="flex gap-[6px]">
+              <Image src="/icons/location-red.svg" alt="위치아이콘" width={16} height={16} />
+              <div className="text-sm font-normal leading-snug text-zinc-500 ">{storeData.address1}</div>
+            </div>
+            <p className="leading-[26px] text-[14px] md:text-[16px] line-clamp-3">{storeData.description}</p>
           </div>
-          <div className="flex gap-[6px]">
-            <Image src="/icons/location-red.svg" alt="위치아이콘" width={16} height={16} />
-            <div className="text-sm font-normal leading-snug text-zinc-500 ">{storeData.address1}</div>
+          <div className="flex gap-2 ">
+            <Link href={`/my-store/${storeId}/edit`} className="w-full">
+              <Button background="bg-white" className="py-2 font-bold font-pretendard">
+                편집하기
+              </Button>
+            </Link>
+            <Link href={`/my-store/${storeId}/my-recruits/edit`} className="w-full">
+              <Button background="bg-primary" className="py-2 font-bold">
+                공고 등록하기
+              </Button>
+            </Link>
           </div>
-          <p className="leading-[26px] text-[14px] md:text-[16px] line-clamp-3">{storeData.description}</p>
-        </div>
-        <div className="flex gap-2 ">
-          <Link href={`/my-store/${storeId}/edit`} className="w-full">
-            <Button background="bg-white" className="py-2 font-bold font-pretendard">
-              편집하기
-            </Button>
-          </Link>
-          <Link href={`/my-store/${storeId}/my-recruits/edit`} className="w-full">
-            <Button background="bg-primary" className="py-2 font-bold">
-              공고 등록하기
-            </Button>
-          </Link>
         </div>
       </div>
-    </div>
+      {count === 0 ? (
+        <RegisteredNotice />
+      ) : (
+        <HaveRegisteredNotice noticeData={noticeData} storeData={storeData} storeId={storeId} />
+      )}
```

Comment:
P5: 이렇게도 써볼 수 있을 것 같습니당

```suggestion
      {count ? (
        <HaveRegisteredNotice noticeData={noticeData} storeData={storeData} storeId={storeId} />
      ) : (
        <RegisteredNotice />
      )}
```